src/wp-includes/html-api/class-wp-html-tag-processor.php

```diff
@@ -1126,6 +1138,7 @@ public function paused_at_incomplete_token(): bool {
 	 *     // Outputs: "free <egg> lang-en "
 	 *
 	 * @since 6.4.0
+	 * @since 6.7.0 Class names are no longer force lower-cased.
```

Comment:
How about something affirmative like…

```php
/**
 * @since 6.7.0 Class name casing corresponds to document compatibility mode.
 */
```

Reply:
I'm open to changing this, but if it's included I think it should be clear what's changed.

Maybe this is a bugfix and does not need a since annotation at all.

Reply:
I've removed the since tag.